src/main/java/at/hannibal2/skyhanni/features/combat/FerocityDisplay.kt

```diff
@@ -20,16 +21,18 @@ object FerocityDisplay {
      */
     private val ferocityPattern by RepoPattern.pattern(
         "combat.ferocity.tab",
-        " Ferocity: §r§c⫽(?<stat>.*)"
+        " Ferocity: §r§c⫽(?<stat>.*)",
     )
 
     private var display = ""
 
     @SubscribeEvent
-    fun onTabListUpdate(event: TabListUpdateEvent) {
+    fun onTabListUpdate(event: WidgetUpdateEvent) {
```

Comment:
why do we only check for stats widget for resetting, not for the regex?

Reply:
idk, what you mean.

Reply:
`event.lines.matchFirst(ferocityPattern) {`
this happens for every TabWidget update, not only for `TabWidget.STATS` and `TabWidget.DUNGEON_SKILLS_AND_STATS`

Reply:
oh, I missed the return